package.json

```diff
@@ -116,7 +121,10 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "lint:fix": "eslint --fix --"
+    "lint:fix": "eslint --fix --",
+    "extract-messages": "NODE_ENV=production babel ./src  --out-file /dev/null",
```

Comment:
output to a specific file instead of ``/dev/null`(terminal)?

Reply:
@frankobe it's already output to files, it won't output to terminal